README.md

```diff
@@ -351,152 +352,185 @@ or equal to those of the children and the lowest key is in the root node
+ - [README.md](LeetCode/LinkedList/README.java)
+ - [reverseLinkedList](LeetCode/LinkedList/reverseLinkedList.java)
+
+### Queue
+
+ - [movingAverageFromDataStream](LeetCode/Queue/movingAverageFromDataStream.java)
+ - [README.md](LeetCode/Queue/README.java)
+
+### Sort
+
+ - [meetingRoomsII](LeetCode/Sort/meetingRoomsII.java)
+ - [meetingRooms](LeetCode/Sort/meetingRooms.java)
+
+### Stack
+
+ - [binarySearchTreeIterator](LeetCode/Stack/binarySearchTreeIterator.java)
+ - [decodeString](LeetCode/Stack/decodeString.java)
+ - [flattenNestedListIterator](LeetCode/Stack/flattenNestedListIterator.java)
+ - [trappingRainWater](LeetCode/Stack/trappingRainWater.java)
+
+### String
+
+ - [addBinary](LeetCode/String/addBinary.java)
+ - [countAndSay](LeetCode/String/countAndSay.java)
+ - [decodeWays](LeetCode/String/decodeWays.java)
+ - [editDistance](LeetCode/String/editDistance.java)
+ - [integerToEnglishWords](LeetCode/String/integerToEnglishWords.java)
+ - [longestPalindrome](LeetCode/String/longestPalindrome.java)
+ - [longestSubstringWithAtMostKDistinctCharacters](LeetCode/String/longestSubstringWithAtMostKDistinctCharacters.java)
+ - [minimumWindowSubstring](LeetCode/String/minimumWindowSubstring.java)
+ - [multiplyString](LeetCode/String/multiplyString.java)
+ - [oneEditDistance](LeetCode/String/oneEditDistance.java)
+ - [palindromePermutation](LeetCode/String/palindromePermutation.java)
+ - [README.md](LeetCode/String/README.java)
+ - [reverseVowelsOfAString](LeetCode/String/reverseVowelsOfAString.java)
+ - [romanToInteger](LeetCode/String/romanToInteger.java)
+ - [validPalindrome](LeetCode/String/validPalindrome.java)
+ - [validParentheses](LeetCode/String/validParentheses.java)
+
+### Tree
+
+ - [binaryTreeMaximumPathSum](LeetCode/Tree/binaryTreeMaximumPathSum.java)
+ - [binaryTreePaths](LeetCode/Tree/binaryTreePaths.java)
+ - [inorderSuccessorInBST](LeetCode/Tree/inorderSuccessorInBST.java)
+ - [invertBinaryTree](LeetCode/Tree/invertBinaryTree.java)
+ - [lowestCommonAncestorOfABinaryTree](LeetCode/Tree/lowestCommonAncestorOfABinaryTree.java)
+ - [sumOfLeftLeaves](LeetCode/Tree/sumOfLeftLeaves.java)
+ - [validateBinarySearchTree](LeetCode/Tree/validateBinarySearchTree.java)
+
+### Trie
+
+ - [addAndSearchWordDataStructureDesign](LeetCode/Trie/addAndSearchWordDataStructureDesign.java)
+ - [implementTrie](LeetCode/Trie/implementTrie.java)
+ - [wordSquares](LeetCode/Trie/wordSquares.java)
+
+### Two Pointers
+ 
+ - [3Sum](LeetCode//3Sum.java)
+ - [3SumSmaller](LeetCode/TwoPointers/3SumSmaller.java)
+ - [mergeSortedArray](LeetCode/TwoPointers/mergeSortedArray.java)
+ - [minimumSizeSubarraySum](LeetCode/TwoPointers/minimumSizeSubarraySum.java)
+ - [moveZeros](LeetCode//moveZeros.java)
+ - [removeDuplicatesFromSortedArray](LeetCode/TwoPointers/removeDuplicatesFromSortedArray.java)
+ - [reverseString](LeetCode/TwoPointers/reverseString.java)
+ - [sortColors](LeetCode/TwoPointers/sortColors.java)
+ 
+## 18 directories, 124 files
```

Comment:
Okay. I did it. Is there anything to change?
Coz I'm going to sleep. You know I come from India and all the folks are sleeping only night watcher are still.
I think you have a nice morning. :)